### PR TITLE
DPD-500 Resolved clientID resetting issue

### PR DESCRIPTION
**Pull request by igorkol**

### CODE REVIEW

File: src/gaTracker/helpers.ts [hunk 1]
```diff
@@ -117,63 +119,29 @@ export const sendPageview = async (): Promise<void> => {
 			page_title,
 			page_location,
 			send_to: getWebPropertyId(),
+			event_callback: () => callbackFromGtag(),
```
Comment (TheGame2500) [inline src/gaTracker/helpers.ts:122]:
```suggestion
			event_callback: fetchAndStoreGtagIds,
```

File: src/gaTracker/helpers.ts [hunk 2]
```diff
@@ -117,63 +119,29 @@ export const sendPageview = async (): Promise<void> => {
...
 		});
 		sendGa4Event('page_view', {
 			page_title,
 			page_location,
 			...ga4EventParams(),
+			event_callback: () => callbackFromGtag(),
```
Comment (TheGame2500) [inline src/gaTracker/helpers.ts:128]:
```suggestion
			event_callback: fetchAndStoreGtagIds,
```

File: src/gaTracker/helpers.ts [hunk 3]
```diff
@@ -117,63 +119,29 @@ export const sendPageview = async (): Promise<void> => {
...
-				}
-			});
-		}
 	}
 	if (hasGA3() && !hasGA4()) {
 		gtag('event', 'page_view', {
 			page_title,
 			page_location,
 			send_to: getWebPropertyId(),
+			event_callback: () => callbackFromGtag(),
```
Comment (TheGame2500) [inline src/gaTracker/helpers.ts:136]:
```suggestion
			event_callback: fetchAndStoreGtagIds,
```

File: src/gaTracker/helpers.ts [hunk 4]
```diff
@@ -117,63 +119,29 @@ export const sendPageview = async (): Promise<void> => {
...
-				}
-			});
-		}
 	}
 	if (hasGA4() && !hasGA3()) {
 		sendGa4Event('page_view', {
 			page_title,
 			page_location,
 			...ga4EventParams(),
+			event_callback: () => callbackFromGtag(),
```
Comment (TheGame2500) [inline src/gaTracker/helpers.ts:144]:
```suggestion
			event_callback: fetchAndStoreGtagIds,
```

**Review by TheGame2500 — CHANGES_REQUESTED** [review]

File: src/gaTracker/helpers.ts (hunk 1 above)
Comment (igorkol) [reply src/gaTracker/helpers.ts:122]: it's intentionally calling a different function as i don't know what we might want to squeeze in there in the future. i don't want to make changes in n different places afterwards :)

**Review by TheGame2500 — APPROVED** [review]

**Pull request merged**